l10n_th_amount_to_text/tests/test_amount_to_text.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo.tests.common import SavepointCase
+
+
+class TestAmountToText(SavepointCase):
+    def test_currency_th_amount_to_text(self):
+        """ verify that amount_to_text converted text to thai language """
+        currency = self.env.ref("base.THB")
+        amount_text_en = currency.amount_to_text(1050.75)
+        self.assertEqual(
+            amount_text_en, "One Thousand And Fifty Baht and Seventy-Five Satang"
+        )
+        currency.with_context({"lang": "th_TH"}).amount_to_text(1050.75)
+        # num2words need version 0.5.7+
+        # amount_text_th = currency.with_context({"lang": "th_TH"}).amount_to_text(
+        #     1050.75
+        # )
+        # self.assertEqual(amount_text_th, "หนึ่งพันห้าสิบบาทเจ็ดสิบห้าสตางค์")
```

Comment:
You comment out test?

Reply:
core odoo used num2words v.0.5.6 (exclude th currency) but this module need num2words v.0.5.7 that I can't test script this function